client/version.go

```diff
@@ -0,0 +1,102 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package client contains business logic of working with pmm-managed.
+package client
+
+import (
+	"context"
+	"os/exec"
+	"regexp"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/percona/pmm/api/agentpb"
+)
+
+const (
+	versionCheckTimeout = 5 * time.Second
+	mysqlBin            = "mysql"
+	xtrabackupBin       = "xtrabackup"
+)
+
+var (
+	mysqlVersionRegexp      = regexp.MustCompile("^.*Ver ([!-~]*).*")
+	xtrabackupVersionRegexp = regexp.MustCompile("^xtrabackup version ([!-~]*).*")
+)
+
+func (c *Client) localMySQLVersion() (string, error) {
```

Comment:
`mysqlClientBinaryVersion`?

Reply:
I could rename it just to `mysqlVersion`. What do you think?

Reply:
What do we check here? Version of MySQL client installed locally, aren't we?

Reply:
I need to check mysql server version instead of client  (`mysqld` instead of `mysql`) 🤦 Thanks for pointing this out!